src/contexts/bridge.context.tsx

```diff
@@ -0,0 +1,44 @@
+import { createContext, FC, useCallback, useContext } from "react";
+
+import { useEnvContext } from "src/contexts/env.context";
+import { Bridge } from "src/domain";
+import * as api from "../adapters/bridge-api";
```

Comment:
Didn't you change your VSCode config to default importing absolute routes?

Reply:
I import it manually I think, because VS Code doesn't support this `import * as api`! That's why it was wrong, fixed!